crates/iota/src/name_commands.rs

```diff
@@ -49,19 +49,30 @@ use crate::{
     key_identity::{KeyIdentity, get_identity_address},
 };
 
+/// The overbid must be at least of 1 IOTA, which is 10^9 NANOs
+const MIN_OVERBID: u64 = 1_000_000_000;
+
 /// Commands related to the auction system
 #[derive(Parser)]
 pub enum AuctionCommand {
     /// Place a new bid
     Bid {
+        /// The full name of the domain. Ex. my-domain.iota
         domain: Domain,
-        amount: u64,
+        /// The initial bid amount. Must be at least the minimum cost of the
+        /// domain. Defaults to the minimum.
```

Comment:
I would also mention the MIN_OVERBID ?

Reply:
I think you just copy pasted the doc from `Start` because saying "initial bid amount" is also wrong